update widget embed options with new features

oriziv opened the pull request:
we added &title= to the iframe url
and &allowEdit for getting editing token, so user can embed with editing enabled

azurecla commented:
Hi __@zhalu__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (zhalu). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

tysonn commented:
@zhalu - thanks for the update.

oriziv committed:
update widget embed options with new features

we added &title= to the iframe url
and &allowEdit for getting editing token, so user can embed with editing enabled
